docs/user/dataModels.md

```diff
@@ -0,0 +1,102 @@
+Overview
+--------
+
+MasterMind uses relational data models to represent the various entities that it tracks. This section provides a full reference to all of them.
+
+Actors
+------
+
+This model represents a MasterMind Actor, which could refer to a singular user or an organization of many users. The Actor credentials are obtained from a Fiware Lab user/organization.
+
+An Actor is defined by:
+
+  - **Fullname**: The full name of the actor
+  - **Email**: The email associated to the actor
+
+Projects
+--------
+
+This model represents a Project within MasterMind, where each Project contains its own set of registered Actors, Services, Clusters and Subscriptions.
+
+A Project is defined by:
+
+  - **Name**: The name of the project
+  - **Description**: The description of the project
+
+Clusters
+--------
+
+This model represents a Docker Swarm Cluster registered to a project, where the IoT Services are going to be deploted onto.
+
+A Cluster is defined by:
+
+  - **Name**: The name MasterMind is going to refer the cluster by
+  - **Description**: A description for the registered cluster
+  - **Project**: The project this cluster is registered to
+  - **Endpoint**: The endpoint of the master node of the Docker Swarm cluster
+  - **Cert**: The Cert certificate for accessing the cluster using TLS
+  - **Ca**: The Ca certificate for accessing the cluster using TLS
+  - **Key**: The Key for accessing the cluster using TLS
+
+Roles
+--------
+
+This model represents a Role a certain Actor has within a project. This can be either an Admin role or a User role, where the latter doesn't have permission to perform certain actions inside of a MasterMind project, such as deleting a project entirely for example.
+
+A Role is defined by:
+
+  - **Role Level**: The Role Level, which can be either 1 for Admin or 2 for User
+  - **Project**: The Project this Role refers to
+  - **Actor**: The Actor this Role is assigned to
+
+Services
+--------
+
+This model represents an IoT Service registered within a MasterMind Project. Services within MasterMind are generally mapped to what Docker Swarm refers to as a Stack, as IoT Services can be made out of multiple separate components that Docker refers to as individual services (e.g. a Context Broker and its Database).
+
+A Service is defined by:
+
+  - **Name**: The name MasterMind is going to refer to the service by
+  - **Service Type**: The Service Type this Service belongs to
+  - **Project**: The Project this service belongs to
+  - **Cluster**: The Cluster this Service is or will be deployed to
+  - **Docker Service ID**: The ID of the deployed Service (stack) on Docker
+  - **Configuration**: The set of environment variables to provide to the Service when deploying it to a Cluster
+  - **Status**: The current status of the Service on the Cluster (e.g. Active, Inactive)
+  - **Endpoint**: The endpoint the Service can be reached at
+  - **Latitude**: The Latitude of the geographical location of this Service
```

Comment:
for the time being i would comment this part and hide in gui as well. let's first it understand what we will do with it ;)

Reply:
@chicco785 Yeah, you're right. Eventually it's probably something that will be automatically set based on the Cluster the Service runs on (or more specifically the node perhaps). For now I'll hide these.